tests/devices/fa/device_fa_test.py

```
@@ -43,6 +43,94 @@ def _setup_device(self) -> None:
             customize="",
         )
 
+    def _make_560000ag_device(self) -> MideaFADevice:
+        """Create a 560000AG test device (MAM060CPAPWH)."""
+        return MideaFADevice(
+            name="MAM060CPAPWH Test Fan",
+            device_id=2,
+            ip_address="192.168.1.2",
+            port=12345,
+            token="AA",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token"

(S106)